net/crypt.ts

```diff
@@ -0,0 +1,332 @@
+/*
+ * [js-sha1]{@link https://github.com/emn178/js-sha1}
+ *
+ * @version 0.6.0
```

Comment:
I think it would be better to call this file sha1.ts ?

Could you add a very minimal test file called sha1_test.ts - just so we can see it exercised independently

Reply:
done. renamed to `sha1.ts`, but I wonder to include ported codes from npm into deno's standard library. I added this temporaliry because there no way to generate sha1 hash in deno. If deno supports native hash function, this code should gracefully removed with honor to original author.